styles.css

```diff
@@ -132,3 +133,424 @@ ul {
+  border-radius: 4px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.js {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px 12px;
+  width: 58px;
+  height: 16px;
+  background: #ebf0ee;
+  border-radius: 4px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.html {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px 12px;
+  width: 27px;
+  height: 16px;
+  background: #ebf0ee;
+  border-radius: 4px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.button-space {
+  width: 305px;
+  height: 101px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin: auto;
+  text-align: center;
+  display: flex;
+  align-items: center;
+}
+
+.project-text {
+  height: 48px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  background: #36b37e;
+  padding: 12px;
+  border: none;
+  border-radius: 4px;
+  margin: auto;
+}
+
+.the-rest {
+  padding-left: 25px;
+}
+
+.about {
+  background-image: url("Images/Aboutme-background\ \(1\).png"), url("Images/illustration.png");
+  background-position: left top, right top;
+  background-repeat: no-repeat;
+}
+
+.gap {
+  width: 100%;
+  height: 15px;
+}
+
+.illustration {
+  width: 68.46px;
+  height: 107.94px;
+  float: right;
+}
+
+.about-head {
+  margin: 40px auto 0;
+  margin-left: 60px;
+  display: block;
+  align-items: center;
+  justify-content: center;
+  height: 48px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.about-text {
+  width: 355px;
+  height: 189px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.resume-text {
+  height: 48px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  background: #36b37e;
+  padding: 12px;
+  border: none;
+  border-radius: 4px;
+  margin: auto;
+}
+
+button:hover {
+  border-radius: 4px;
+  background: #36b37f;
+}
+
+button:active {
+  border-radius: 4px;
+  background: #008552;
+}
+
+.works1 {
+  padding-top: 60px;
+  margin: auto;
+  display: block;
+  height: 1127px;
+}
+
+.resume-block {
+  width: 100%;
+  height: 48px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin: auto;
+  text-align: center;
+  display: flex;
+}
+
+.container {
+  height: 352px;
+  background: #ebf0ee;
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 24px;
+}
```

Comment:
Kindly make sure when you fix the background image in the about section the borders should be still present in the technologies cards. :eyes: 
![image](https://user-images.githubusercontent.com/98363075/192098129-4e05ea65-d231-457c-b28a-61b483f9947d.png)